ompi/request/req_test.c

```diff
@@ -321,30 +340,42 @@ int ompi_request_default_test_some(
     int * indices,
     ompi_status_public_t * statuses)
 {
-    size_t i, num_requests_null_inactive = 0, num_requests_done = 0;
```

Comment:
?

Reply:
That was a leftover from using goto earlier, fixed.